content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+
+
+**Properties:**
+
+
+  * `rules`
+    List of FirewallRules which will form this SecurityGroup. Only the `sources:` and `allowed:` fields should be supplied (see FirewallRule properties for details).
+
+    *default:* []
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+  * `name`
+    Optional security group name. By default it will be network name plus node name.
+
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.SslCertificate
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A TLS/SSL certificate and key. This will be used by a HTTPS TargetProxy to provide authenticated encryption for connecting users.
+
+
+
+**Properties:**
+
+
+  * `private_key`
+    Dictionary describing private key in PEM format used to generate this SSL certificate. Expected format is:
+      type: text|file
+      data: Private key in PEM format if text, otherwise path to a file with private key
+
+    *default:* {}
+  * `name`
+    Optional SSL certificate name. By default it will be SSL certificate id.
+
+    *type:* string
+    *default:*
+  * `certificate`
+    Certificate (self-signed or obtained from CA) in PEM format. Expected format is:
+      type: text|file
+      data: Certificate in PEM format if text, otherwise path to a file with certificate
+
+    *default:* {}
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Additional setting for target proxy
+
+    *default:* {}
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.StaticIP
+**Derived From:** cloudify.gcp.nodes.GlobalAddress
+
+Alias for GlobalAddress for backward compatibility.
+
+
+
+**Properties:**
+
+
+
+
+
+
+
+## cloudify.gcp.nodes.SubNetwork
+**Derived From:** [cloudify.nodes.Subnet]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP Subnetwork. Must be connected to a Network using `cloudify.gcp.relationships.contained_in_network`.
+
+Only networks with the `auto_subnets` property disabled can be used.
+
+
+
+**Properties:**
+
+
+  * `subnet`
+    The subnet, denoted in CIDR form (i.e. '10.8.0.0/20') Subnets must be unique and non-overlapping within a project. See https://cloud.google.com/compute/docs/subnetworks#networks_and_subnetworks
+
+    *type:* string
+    *default:*
+  * `region`
+    The region this subnet is in. See https://cloud.google.com/compute/docs/regions-zones/regions-zones
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
```

Comment:
Indicate**s** whether the resource exists **and should be used (true),** or if Cloudify should create the resource **(false).**